apps/website/src/components/shared/form/CheckboxField.tsx

```diff
@@ -12,7 +13,9 @@ export function CheckboxField(props: CheckboxFieldProps) {
   const { inputProps } = useCheckbox(props, state, ref);
 
   return (
-    <label className="flex flex-row items-start gap-3">
+    <label
+      className={`flex flex-row items-start gap-3 ${props.className || ""}`}
```

Comment:
Should we be using the classes helper here, so we're not leaving a trailing space if className is empty?

Reply:
tbh i dont care about the trailing space, but i used the helper anyways